<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EU-WALL Main</string>
  </property>
  <widget class="QFrame" name="mainFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#mainFrame {
background-color: qlineargradient(
    spread:reflect,
    x1:0, y1:0,
    x2:1, y2:1,
    stop:0 rgba(148, 4, 3, 255),
    stop:1 rgba(239, 174, 20, 255)
);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="mainTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>801</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 28pt &quot;Futura&quot;; color: white;</string>
    </property>
    <property name="text">
     <string>EU-WALL: Feed</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; border-radius: 10px; font: 11pt &quot;Arial&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Search posts by keyword or date (e.g. 2025-05-15)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; background-color: black; border-radius: 10px; font: 11pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="myPostsButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; background-color: black; border-radius: 10px; font: 11pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>My Posts</string>
    </property>
   </widget>
   <widget class="QListWidget" name="postListWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>701</width>
      <height>230</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="newPostTextEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>701</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; border-radius: 10px; font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Write your post here...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="anonymousCheckBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Post Anonymously</string>
    </property>
   </widget>
   <widget class="QPushButton" name="postButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>440</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; background-color: black; border-radius: 15px; font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Post</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
